common/source/docs/common-rtf.rst

```diff
@@ -90,19 +84,23 @@ Subs from Partners
 
 * `Blue Robotics - BlueROV2 <https://bluerobotics.com/store/rov/bluerov2/>`__
 
-
 Vehicles from Non-Partners
 ==========================
 
 * `Aton <https://traxxas.com/products/models/heli/Aton-Plus>`__ and `Aton-Plus from traxxas <https://traxxas.com/products/models/heli/Aton-Plus>`__ (firmware loaded using an SD Card)
-* DRONEE  Easy to Use Mapping Plane Drone `DRONEE PLANE <https://dronee.aero/pages/droneeplane>`__
+* `BathyDrone mapping boats <https://www.bathydrone-usv.com/>`__
+* `DRONEE Easy to Use Mapping Plane <https://dronee.aero/pages/droneeplane>`__
 * `MotoDoro Farm Mapper (Plane) <https://motodoro.com/blog/detail/00005-farm-mapper-vtol.html>`__
 * `SkyRocket - Journey <http://sky-viper.com/journey/>`__
+* `TT Robotix - CX-180 ICEMAN Helicopter <http://www.ttrobotix.com/products/detail/925.html>`__
```

Comment:
I think we would need to add these back to the Partners list?

Reply:
yes, sorry, very careless of me.  Not enough coffee yet.  Thanks!